.github/workflows/run-terraform.yml

```diff
@@ -593,7 +593,7 @@ jobs:
 
   attest-deploy:
     if: inputs.image_url != '' && (inputs.environment == 'dev' || inputs.environment == 'test')
-    name: Attest deploy to environment
+    name: Attest deploy to ${{ inputs.environment }}
```

Comment:
Tror vi må bruke en temp variabel i mellom her for å unngå direkte inputs.
Se denne: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable
Issue:  https://kartverket.atlassian.net/jira/software/projects/SKIP/boards/2/backlog?selectedIssue=SKIP-642

Reply:
Tror kanskje det går fint såfremt det ikke er i en run kommando?

Reply:
Ser ut som at @anderssonw holder på å fikse det for resterende (bortsett fra run-security-scans) i PR https://github.com/kartverket/github-workflows/pull/22. 
Slik jeg skjønte det så trenger vi kun oppbevare inputs i `env` dersom de skal brukes i en `run` bolke.. pls correct me if I'm wrong 🙏

Reply:
Jeg og @aa-kv snakket litt, og tenker at den greieste løsningen er rett og slett at vi konsekvent bruker en midlertidig env var, da det gjør at vi ikke plutselig går på en smell, og vi ikke må tenke oss om på hva vi faktisk burde bruke.

Reply:
da skal det være fikset i run-security-scans.yml 😄 Ser ut til at https://github.com/kartverket/github-workflows/pull/22 tar for seg å fikse det i run-terraform.yml og post-build-attest.yml